registries/adapters/apiv2_adapter.go

```diff
@@ -0,0 +1,245 @@
+//
+// Copyright (c) 2018 Red Hat, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package adapters
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/automationbroker/bundle-lib/bundle"
+	"github.com/automationbroker/bundle-lib/registries/adapters/oauth"
+	log "github.com/sirupsen/logrus"
+)
+
+const apiV2ManifestURL = "%v/v2/%v/manifests/%v"
+const apiV2CatalogURL = "%v/v2/_catalog"
+
+// OpenShiftAdapter - OpenShift Adapter
+type OpenShiftAdapter struct {
+	*apiV2Adapter
```

Comment:
There a specific reason these are getting embedded as pointers?

Reply:
@eriknelson no, I was just following the recent example that @mhrivnak added to the [rhcc_adaptor](https://github.com/automationbroker/bundle-lib/blob/master/registries/adapters/rhcc_adapter.go#L31)
I don't have a real good reason for it, I will remove it